docs/source/deploy.rst

```diff
@@ -267,7 +261,7 @@ Either reboot, or start the services manually::
     systemctl start gunicorn.socket
 
 
-After running ``curl --unix-socket /run/gunicorn.sock http``, Gunicorn
+After running ``sudo -u nginx curl --unix-socket /run/gunicorn.sock http``, Gunicorn
```

Comment:
why would not `johndoe` do that?

Reply:
To make sure that the nginx proxy really can connect to the socket - that's what matters.

Reply:
sounds good for me, but maybe you can add a line or two about this username, so the user would know what to change in its own config?